app/assets/javascripts/config/angularRoutes.js.coffee

```diff
@@ -96,6 +100,8 @@
           templateUrl: 'account/templates/create-account.html'
           controller: 'AccountController'
       onEnter: ['AccountService', (AccountService) ->
+        AccountService.clearAccountMessages()
```

Comment:
I wonder for the places where it says

```
AccountService.clearAccountMessages()
AccountService.resetUserAuth() 
````

if it makes sense to do at stateChangeStart, and use the toState.name to map through these routes, and if it matches, we call these so there is less redundancy.